### PR TITLE
ci(): changelog action + merge directive

### PR DESCRIPTION
**Pull request by ShaMan123**
The changelog action is failing due to an unknown reason on #8135 , see [failing job](https://github.com/fabricjs/fabric.js/actions/runs/3173069251/jobs/5168265853) and [CHANGELOG](https://github.com/fabricjs/fabric.js/pull/8135/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed)

This PR removes the action and adds a simple diff command to check if the file has changed instead of the action

In addition added `.gitattribute` instructing git to do a merge union in case of conflicts in CHANGLOG.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -56,15 +56,15 @@ jobs:
   changelog:
     if: ${{ always() }}
```
Comment (ShaMan123) [inline .github/workflows/build.yml:57]: not sure why this is needed

**Review by ShaMan123 — COMMENTED** [review]
READY

**Comment by ShaMan123** [comment]
@asturur what if we disable the lint action for now?
So we can see the green tick mark of all passing and then reinstate it in the end of the migration

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# https://stackoverflow.com/a/13264117/9068029
+CHANGELOG.md merge=union
```
Comment (ShaMan123) [inline .gitattributes:2]: This is a cool thing instructing git to do a merge union in case of conflicts

**Review by ShaMan123 — COMMENTED** [review]
ready

File: .gitattributes (hunk 2 above)
Comment (ShaMan123) [reply .gitattributes:2]: tested it in this PR when I ran update from master and indeed it fast forwarded and merged the file as expected

**Comment by asturur** [comment]
Why i don't see action starting here?

**Review by asturur — APPROVED** [review]

**Pull request merged**